web/core/src/jsMain/kotlin/androidx/compose/web/DomApplier.kt

```diff
@@ -45,16 +46,31 @@ open class DomNodeWrapper(open val node: Node) {
         val htmlElement = node as? HTMLElement ?: return
 
         currentListeners.forEach {
-            htmlElement.removeEventListener(it.event, it)
+            htmlElement.removeEventListener(it.event, it, it.options.toJsObject())
         }
 
         currentListeners = list
 
         currentListeners.forEach {
-            htmlElement.addEventListener(it.event, it)
+            htmlElement.addEventListener(it.event, it, it.options.toJsObject())
         }
     }
 
+    private fun Options.BooleanValue.toBoolean(): Boolean? = when (this) {
+        Options.BooleanValue.True -> true
+        Options.BooleanValue.False -> false
+        Options.BooleanValue.Default -> null
+    }
+
+    // This conversion helps to preserve the default values of options parameters (defaults can vary per browser)
+    private fun Options.toJsObject(): dynamic {
+        val obj = js("{}")
```

Comment:
we have `jsObject` util, use it and put into appropriate place.
common use-case 
```Kotlin
jsObject<SomeParams/* external interface*/> {
  param1 = ...
  param2 = ...
}
```

Reply:
great! thanks, I'll use it

Reply:
fixed this